src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -9355,9 +9359,9 @@ void VRFTerminalUnitEquipment::ControlVRFToLoad(EnergyPlusData &state,
         }
     } else if (VRFCoolingMode || HRCoolingMode) {
         // IF the system is in cooling mode and/or the terminal unit requests cooling
-        if (NoCompOutput <= QZnReq) {
-            PartLoadRatio = 0.0;
```

Comment:
Are you sure  PLR=0 is no longer needed here?

Reply:
Well, I did not need it for the tests I have run. Maybe need if there are no supplemental heating coil.

Reply:
OK. I modified the code to include `PLR=0` as follows:

```
        if ((NoCompOutput <= QZnReq && QZnReq < -DataHVACGlobals::SmallLoad) ||
            (QZnReq >= -DataHVACGlobals::SmallLoad && NoCompOutput <= QZnReq && !HRCoolingMode)) {
            state.dataHVACVarRefFlow->VRFTU(VRFTUNum).coolingCoilActive = false;
            if (!this->SuppHeatingCoilPresent) {
                PartLoadRatio = 0.0;
                return;
            }
        }
```